docs/snippets/web3ToProvider.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Getting Accounts
+ */
+
+// window.web3
+const accounts = web3.eth.accounts;
+
+// window.ethereum
+const accounts = await ethereum.request({ method: 'eth_accounts' });
+
+/**
+ * Sending a Transaction
+ */
+
+// window.web3
+web3.eth.sendTransaction(
+  {
+    to: '0x...',
+    'from': '0x...',
```

Comment:
Is there some reason this one is quoted? Similarly for the `request` example

Reply:
Syntax highlighting 😞